Content.Server/Chemistry/Components/InjectorComponent.cs

```diff
@@ -123,6 +139,13 @@ async Task<bool> IAfterInteract.AfterInteract(AfterInteractEventArgs eventArgs)
 
             var targetEntity = eventArgs.Target;
 
+            // is the target a mob? If yes, add a use delay
+            if (Owner.EntityManager.HasComponent<MobStateComponent>(targetEntity.Uid) ||
```

Comment:
Hm, this brings up the question of whether this should also apply to dead or crit and whether the delay should happen for injecting yourself. Maybe just a shorter timer.

Reply:
I'll make self-inject or incapacitated mob delay half (or a third?) as long. That seems reasonable.
Injecting into a mob is still more complicated than just adding liquid to a beaker, so there should probably still be some sort of do-after.

Reply:
Changed:
- Added 1-second lower bound on the **base** do-after delay
- made self-inject & incapacitated take half as long (so min of 0.5 seconds)
- combat mode adds +1 second to prevent cheese injection with a fast syringe & lag.